### PR TITLE
update(Gunshop): Added Revolver Back

### LINKED ISSUE #1409: re add revolver
it was disabled since it was a 1 shotter so people felt it was "too OP"

now that we can change damage modifiers for weapons we should re add it and just adjust it so it's like 2-3 hits instead so we can still have the dope revolver

### PR DESCRIPTION
**Pull request by Savage912-Official**
Allow the revolver to be purchased again, and it now takes 3 shots to kill someone rather than 1 shot.

closes #1409

### CODE REVIEW

**Review by minipunch — APPROVED** [review]

**Comment by minipunch** [comment]
Actually now that I think of this

Instead of just buying it from the gun store

We should make this exclusively craftable as an incentive for blacksmiths

File: resources/usa-crafting/config.lua [hunk 1]
```diff
@@ -192,6 +192,26 @@ Config.recipes = {
             requiredCraftingLevel = 1,
             type = "weapons"
         },
+        { 
+            name = "Revolver",
+            image = "https://wiki.gtanet.work/images/a/a2/weapon_-1045183535.png",
```
Comment (minipunch) [inline resources/usa-crafting/config.lua:197]: U sure this image works?

I can time out error when connecting to that link

can u use imgur instead

**Review by minipunch — APPROVED** [review]

**Review by minipunch — CHANGES_REQUESTED** [review]
imgur link comment

File: resources/usa-crafting/config.lua (hunk 1 above)
Comment (Savage912-Official) [reply resources/usa-crafting/config.lua:197]: I'll change it but yeah it worked in game

**Review by minipunch — APPROVED** [review]

**Pull request merged**